modules/openid_auth/app/controllers/openid_auth/v2/validation_controller.rb

```diff
@@ -81,6 +86,28 @@ def setup_structure
         end
         payload_object
       end
+
+      def charon_token_screen?(payload_object)
+        act_vid = payload_object.act[:vista_id]
```

Comment:
This is just a preference so take it or leave it, but i think vista_id is short enough where the abbreviation to vid is not necessary.

Reply:
Will change to make more readable.